apps/server/cypress/e2e/suite2/sophtron.cy.ts

```diff
@@ -63,5 +63,9 @@ describe("Sophtron aggregator", () => {
     expectConnectionSuccess();
   });
 
-  generateDataTests({ makeAConnection, shouldTestVcEndpoint: true });
+  generateDataTests({
+    makeAConnection,
+    shouldTestVcEndpoint: true,
+    queryString: "?start_time=2021/1/1&end_time=2022/1/1",
```

Comment:
This variable name makes it unclear where this is going to be used. Maybe transactionsQueryString?

Reply:
Renaming done